src/store/clueSlice.js

```diff
@@ -3,6 +3,7 @@ import { createSlice } from '@reduxjs/toolkit';
 const initialState = {
   currentClue: {},
   clueHistory: [],
+  guessesRemaining: 0,
```

Comment:
just wanted to note that this is guessesRemaining for the currentClue (maybe that's obvious) and this is almost like a 'counter' that gets decremented , right?